flake.nix

```diff
@@ -175,8 +172,9 @@
       in
         lib.recursiveUpdate flake rec {
           project = cabalProject;
-          # add a required job, that's basically all hydraJobs.
-          hydraJobs =
+          # skip aarch64-linux as we don't have any native builders for this arch as of 2024-07-15
+          # so this would choke the CI. aarch64-linux binary building is enabled through cross-compilation.
```

Comment:
A little foreshadowing for: https://github.com/IntersectMBO/cardano-cli/pull/686#issuecomment-2033090278